src/box/key_def.h

```diff
@@ -731,6 +731,8 @@ key_def_decode_parts(struct key_part_def *parts, uint32_t part_count,
 
 /**
  * Returns the part in index_def->parts for the specified fieldno.
```

Comment:
Nit: if we don't reject json path here, we should write something like "returns the first part in `index_def->parts` for the specified fieldno" since we can have several parts indexing one field with different json paths.

Reply:
Done.